### PR TITLE
After the API update that restores the correct order of directories i…

### PR DESCRIPTION
**Pull request by EstherDarkish**
…n path, we have to undo the "reverse" that was previously done.

### CODE REVIEW

File: src/components/graph/menus/network-modification-node-editor.jsx [hunk 1]
```diff
@@ -764,31 +764,39 @@ const NetworkModificationNodeEditor = () => {
...
+        const parentDirectoryIndex = studyPath.length - 2; // Should always be the second to last element
+
+        const studyName = studyPath[studyIndex].elementName;
+
+        const studyDirectoryPath =
             '/' +
             studyPath
-                .slice(PARENT_DIRECTORY_INDEX)
-                .reverse()
+                .filter((r) => r.type === ElementType.DIRECTORY)
```
Comment (antoinebhs) [inline src/components/graph/menus/network-modification-node-editor.jsx:784]: In another place (https://github.com/gridsuite/gridstudy-app/blob/de6b1aee232bbc492c027999a56f38c297034711/src/components/study-container.jsx#L390C28-L390C51), we use slice as before, keep it for consistency ?
```suggestion
                .slice(0, studyPath.length - 1)
```

**Review by antoinebhs — APPROVED** [review]
Code OK
Test OK
Small suggestions for consistency with other uses of fetchDirectoryElementPath

File: src/components/graph/menus/network-modification-node-editor.jsx [hunk 2]
```diff
@@ -797,14 +805,16 @@ const NetworkModificationNodeEditor = () => {
...
-            headerId: 'infoCreateModificationsMsg',
-            headerValues: {
-                nbModifications: selectedItems.length,
-                studyDirectory: studyDirName,
-            },
-        });
-        createModifications(studyDirectoryUuid, modificationPropsList)
+        createModifications(studyParentDirectoryUuid, modificationPropsList)
+            .then(() => {
+                snackInfo({
```
Comment (sBouzols) [inline src/components/graph/menus/network-modification-node-editor.jsx:810]: If moved in then, maybe we should change the message `infoCreateModificationsMsg` to be :
```diff
-"infoCreateModificationsMsg": "Creation of {nbModifications} network modifications in {studyDirectory}"
+"infoCreateModificationsMsg": "{nbModifications} network modifications successfully created in {studyDirectory}"
```
fr:
```diff
-"infoCreateModificationsMsg": "Création de {nbModifications} modifications dans {studyDirectory}",
+"infoCreateModificationsMsg": "{nbModifications} modifications dans {studyDirectory} créée(s) avec succès",
```

**Review by sBouzols — APPROVED** [review]
Code Review OK
Test OK
Console warning check OK
a minor translation suggestion

**Pull request merged**